src/lib/components/particles/half-line-text/index.jsx

```diff
@@ -0,0 +1,12 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const HalfLineText = ({ text, height, styles }) => {
```

Comment:
Arguably this component does not need to exist, since all it does is render some text.